src/app/containers/MediaPageMain/index.jsx

```diff
@@ -7,15 +7,38 @@ import ATIAnalytics from '../ATIAnalytics';
 import MetadataContainer from '../Metadata';
 import { Grid, GridItemConstrainedMedium } from '../../lib/styledGrid';
 import { ServiceContext } from '../../contexts/ServiceContext';
+import aresBlockPropTypes from '../../models/propTypes/mediaPage/content/aresBlock';
+
+const renderBlock = ({ block, script, service, key }) => {
```

Comment:
I think this is actually a react component and not a render function because you're passing in essentially a prop argument.

You could go for something like this and then remove the propTypes.

```jsx
const renderBlock = (script, service, key) => block => {

// and then use it like so...

{blocks.map(renderBlock(script, service, key))}
```

either that or I think it should be renamed to `Block` instead of `renderBlock`

Reply:
Good spot @jroebu14, I'll open a follow up PR - it'll be much nicer without the propTypes. ⭐️